File: Respostas/Bee_1065/1065.c
#include <stdio.h>

int main() {
	int number, par = 0;
	
	for (int i = 0; i < 5; i++) {
		scanf("%d", &number);
		if (number % 2 == 0) par++;
	}
	
	printf("%d valores pares\n", par);
	
	return 0;
}

File: Respostas/Bee_1065/1065.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
	public static void main(String[] args) throws IOException {
		int number, par = 0;
		
		Scanner input = new Scanner(System.in);
		
		for (int i = 0; i < 5; i++) {
			number = input.nextInt();
			if (number % 2 == 0) par++;
		}
		
		System.out.printf("%d valores pares\n", par);
	}
}